pandas/core/generic.py

```diff
@@ -478,8 +478,9 @@ def _validate_dtype(cls, dtype) -> DtypeObj | None:
     # ----------------------------------------------------------------------
     # Construction
 
+    # error: Signature of "_constructor" incompatible with supertype "PandasObject"
     @property
-    def _constructor(self) -> Callable[..., Self]:
+    def _constructor(self) -> Callable[..., Self]:  # type: ignore[override]
```

Comment:
Should technically be `type[Self]` but the NDFrame constructor allows far fewer arguments than the Series/DataFrame constructor, which leads to many errors when `_constructor()(...)` is called inside NDFrame, which are at runtime only reached by sub-classes.